rpc/namespaces/eth/utils.go

```diff
@@ -0,0 +1,166 @@
+package eth
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/cosmos/ethermint/x/evm/types"
+
+	"github.com/ethereum/go-ethereum/core/vm"
+)
+
+const (
+	DefaultEVMErrorCode          = -32000
+	VMExecuteException           = -32015
+	VMExecuteExceptionInEstimate = 3
+
+	RPCEthCall           = "eth_call"
+	RPCEthEstimateGas    = "eth_estimateGas"
+	RPCEthGetBlockByHash = "eth_getBlockByHash"
+
+	RPCUnknowErr = "unknow"
```

Comment:
"unknown" ?

Reply:
for now, there only one case need to catch which is "execution reverted", during we test, we haven't catch any another error but not sure that will not happen in future, so `unknow` is a reserved value for future

Reply:
i meant shouldnt it be labeled "unknown" instead of "unknow"

Reply:
ha, nice catch